object-store-client-common/src/main/scala/uk/gov/hmrc/objectstore/client/model.scala

```diff
@@ -67,3 +67,8 @@ final case class ObjectSummary(
   contentMd5: String,
   lastModified: Instant
 )
+
+object ObjectExpirationPeriod extends Enumeration {
+  type ObjectExpirationPeriod = Value
+  val one_week, one_month, six_months, one_year, seven_years, ten_years = Value
```

Comment:
I think the convention is upper camel case for object names https://docs.scala-lang.org/style/naming-conventions.html